script/build

```diff
@@ -2,12 +2,16 @@
 
 'use strict'
 
+const CONFIG = require('./config')
+
 if (process.argv.includes('--no-bootstrap')) {
   console.log('Skipping bootstrap')
 } else {
   // Bootstrap first to ensure all the dependencies used later in this script
   // are installed.
-  require('./bootstrap')
+  const path = require('path')
+  const childProcess = require('child_process')
+  childProcess.execFileSync(process.execPath, [path.join(CONFIG.scriptRootPath, 'bootstrap')], { env: process.env, cwd: CONFIG.repositoryRootPath, stdio: 'inherit' });
```

Comment:
The underlying code runs `async`. Running this directly requires changing all the code since Nodejs does not support top-level asynchronous operations.

Here, I preferred to keep the code concise and clean.